javascript-functions-practice/get-first-character.js

```diff
@@ -0,0 +1,5 @@
+/* exported getFirstCharacter */
+function getFirstCharacter(string) {
+  const firstCharacter = string['0'];
```

Comment:
Instead of passing a string of `'0'` here, I would pass a number. The reason why this works currently, is that JavaScript converts all strings to numbers, but we want to be as specific as possible in our code, so I would just put the number directly.